docs/topics/db/aggregation.txt

```diff
@@ -138,28 +140,32 @@ by providing that name when you specify the aggregate clause::
     {'average_price': 34.35}
 
 If you want to generate more than one aggregate, you just add another
-argument to the ``aggregate()`` clause. So, if we also wanted to know
+argument to the :meth:`~django.db.models.query.QuerySet.aggregate` clause. So, if we also wanted to know
 the maximum and minimum price of all books, we would issue the query::
 
     >>> from django.db.models import Avg, Max, Min
     >>> Book.objects.aggregate(Avg('price'), Max('price'), Min('price'))
     {'price__avg': 34.35, 'price__max': Decimal('81.20'), 'price__min': Decimal('12.99')}
 
-Generating aggregates for each item in a ``QuerySet``
-=====================================================
+Generating aggregates for each item in a :class:`~django.db.models.query.QuerySet`
+==================================================================================
```

Comment:
So I think just use backticks in titles, with cross references in the body text.